from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(626, 335)
        self.centralWidget = QtWidgets.QWidget(MainWindow)
        self.centralWidget.setMaximumSize(QtCore.QSize(16777215, 335))
        self.centralWidget.setObjectName("centralWidget")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.centralWidget)
        self.verticalLayout.setContentsMargins(11, 11, 11, 11)
        self.verticalLayout.setSpacing(6)
        self.verticalLayout.setObjectName("verticalLayout")
        self.horizontalLayout = QtWidgets.QHBoxLayout()
        self.horizontalLayout.setSpacing(6)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.new_btn = QtWidgets.QPushButton(self.centralWidget)
        self.new_btn.setObjectName("new_btn")
        self.horizontalLayout.addWidget(self.new_btn)
        self.open_btn = QtWidgets.QPushButton(self.centralWidget)
        self.open_btn.setObjectName("open_btn")
        self.horizontalLayout.addWidget(self.open_btn)
        self.save_btn = QtWidgets.QPushButton(self.centralWidget)
        self.save_btn.setObjectName("save_btn")
        self.horizontalLayout.addWidget(self.save_btn)
        spacerItem = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem)
        self.author_cbox = QtWidgets.QComboBox(self.centralWidget)
        self.author_cbox.setObjectName("author_cbox")
        self.author_cbox.addItem("")
        self.horizontalLayout.addWidget(self.author_cbox)
        spacerItem1 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem1)
        self.temperature_label = QtWidgets.QLabel(self.centralWidget)
        self.temperature_label.setObjectName("temperature_label")
        self.horizontalLayout.addWidget(self.temperature_label)
        self.temperature_slider = QtWidgets.QSlider(self.centralWidget)
        self.temperature_slider.setOrientation(QtCore.Qt.Horizontal)
        self.temperature_slider.setObjectName("temperature_slider")
        self.horizontalLayout.addWidget(self.temperature_slider)
        self.verticalLayout.addLayout(self.horizontalLayout)
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_2.setSpacing(6)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.text_edit = QtWidgets.QTextEdit(self.centralWidget)
        self.text_edit.setObjectName("text_edit")
        self.horizontalLayout_2.addWidget(self.text_edit)
        self.words_list_widget = QtWidgets.QListWidget(self.centralWidget)
        self.words_list_widget.setMaximumSize(QtCore.QSize(111, 16777215))
        self.words_list_widget.setObjectName("words_list_widget")
        self.horizontalLayout_2.addWidget(self.words_list_widget)
        self.verticalLayout.addLayout(self.horizontalLayout_2)
        MainWindow.setCentralWidget(self.centralWidget)
        self.statusBar = QtWidgets.QStatusBar(MainWindow)
        self.statusBar.setObjectName("statusBar")
        MainWindow.setStatusBar(self.statusBar)

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.new_btn.setText(_translate("MainWindow", "Новый"))
        self.open_btn.setText(_translate("MainWindow", "Открыть"))
        self.save_btn.setText(_translate("MainWindow", "Сохранить как"))
        self.author_cbox.setItemText(0, _translate("MainWindow", "Толстой"))
        self.temperature_label.setText(_translate("MainWindow", "Температура: 0.0"))


